driver-core/src/test/unit/com/mongodb/internal/connection/DefaultConnectionPoolSpecification.groovy

```diff
@@ -152,7 +152,7 @@ class DefaultConnectionPoolSpecification extends Specification {
         connectionFactory.createdConnections.get(0).opened()  // if the first one is opened, they all should be
 
         when: 'the pool is invalidated and the maintenance tasks runs'
-        pool.invalidate()
+        pool.invalidate(null)
```

Comment:
Does production code ever pass null as the argument?  Should it be allowed?

Reply:
Yes, we do pass `null` here not only in tests. See `DefaultSdamServerDescriptionManager.handleException`, where `SdamIssue` may be created via `SdamIssue.unspecified` and not have an associated exception.